src/openrct2/world/Map.cpp

```diff
@@ -1334,9 +1337,9 @@ void map_update_tiles()
         }
 
         // Repeat for each 256x256 block on the map
-        for (int32_t blockY = 0; blockY < gMapSize; blockY += 256)
+        for (int32_t blockY = 0; blockY < gMapSize.y; blockY += 256)
```

Comment:
Why use signed integer?

Reply:
TileCoordsXY is made of signed integer fields, so it it avoids needing any casts.